pkg/apis/core/v1beta1/constants/utils.go

```diff
@@ -34,3 +34,12 @@ func getDeprecatedAnnotation(annotations map[string]string, annotationKey, depre
 
 	return val, ok
 }
+
+// GetShootVPADeploymentNames returns the names of all VPA related deployments related to shoot clusters.
+func GetShootVPADeploymentNames() []string {
```

Comment:
Hm, why is this a function? Can't it be a statically defined `[]string` var?

Reply:
> Can't it be a statically defined []string var?

Yes, it can. OTOH, it is supposed to be constant and a public `[]string` var can be mutated from any place which isn't possible if you use a getter `func`. I'm not strongly voting against the var but still wanted to let you know my though. WDYT?

Reply:
Hm, yes, that's a valid argument, of course. Fine then, thanks.